spec/support/table_row_matcher.rb

```diff
@@ -0,0 +1,51 @@
+module TableRowMatchers
+  class SimpleTableRowMatcher < Capybara::RSpecMatchers::Matcher
```

Comment:
This is really cool! Is this a standard thing to do and are there any docs for how to do this (couldn't find much on a quick google)?